### ISSUE TITLE
Syntax highlighting for Swig interface files

### ISSUE THREAD

**Issue by yyokyfo** [cxasm/notepad--#298]
Swig interface files have the extensions `.i` and `.ii`, and could be treated as `C++`.

**Comment by cxasm** [comment]
![image](https://github.com/cxasm/notepad--/assets/42246867/0c1b0e33-cd14-4dc1-86fa-6a2412bf9236)

**Comment by yyokyfo** [comment]
> ![image](https://private-user-images.githubusercontent.com/42246867/293621820-0c1b0e33-cd14-4dc1-86fa-6a2412bf9236.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDQxNjkzOTUsIm5iZiI6MTcwNDE2OTA5NSwicGF0aCI6Ii80MjI0Njg2Ny8yOTM2MjE4MjAtMGMxYjBlMzMtY2QxNC00ZGMxLTg2ZmEtNmEyNDEyYmY5MjM2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMDIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTAyVDA0MTgxNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWJlMmI4NGRlZDYzOTE2YTAyMDhjNGI2ZmFlZmIzNzUxNTEwZDdjOWE1Y2ZlNGFjMzM5YmVmNDc2YTY4MTdhOWImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.gxlzSsjHoCR-lpLp4YBbXA9OTmCHCINFbf35uIXKt20)

Yes, I know about this. What I suggest is something out of the box, no need to customize by the users.
